yt/visualization/tests/test_line_annotation_unit.py

```diff
@@ -1,6 +1,7 @@
-import yt
 import numpy as np
 
+import yt
```

Comment:
ah, so sorry I'm only catching this now... We should avoid importing the whole library in a test file. Can you please replace this with relevant imports as
```python
from yt.loaders import load_uniform_grid
from yt.visualization.plot_window import ProjectionPlot
```
and remove the `yt.` namespace from lines 10 and 20 respectively.
```